openmls/src/group/mls_group/commit_builder.rs

```diff
@@ -252,17 +250,60 @@ impl<'a> CommitBuilder<'a, Initial> {
         self
     }
 
+    /// Adds a proposal to the proposals to be committed.
+    pub fn add_proposal(mut self, proposal: Proposal) -> Self {
```

Comment:
Maybe add that there's `add_proposals` as well to add multiple in one go? Taking an iterator there looks a little odd though. I guess that's how we'd use it internally. But is this what users should use? Wouldn't they rather have a slice `&[Proposals]`?